apps/dashboard/src/components/primitives/tooltip.tsx

```diff
@@ -0,0 +1,71 @@
+import * as React from 'react';
+import * as TooltipPrimitive from '@radix-ui/react-tooltip';
+
+import { cn } from '@/utils/ui';
+import { cva, VariantProps } from 'class-variance-authority';
+
+const TooltipProvider = TooltipPrimitive.Provider;
+
+const Tooltip = TooltipPrimitive.Root;
+
+const TooltipTrigger = TooltipPrimitive.Trigger;
+
+const tooltipContentVariants = cva(
+  `z-50 overflow-hidden  px-3 py-1.5 text-xs  animate-in fade-in-0 zoom-in-95 data-[state=closed]:animate-out data-[state=closed]:fade-out-0 data-[state=closed]:zoom-out-95 data-[side=bottom]:slide-in-from-top-2 data-[side=left]:slide-in-from-right-2 data-[side=right]:slide-in-from-left-2 data-[side=top]:slide-in-from-bottom-2`,
+  {
+    variants: {
+      variant: {
+        default: 'bg-primary text-primary-foreground',
+        light: 'border border-stroke/20 bg-background shadow-xs',
```

Comment:
The tooltip implemented only a light version with the LG size, not sure if I have to invest time in all variants?

Reply:
It's fine for now!